docs/api/en/renderers/webxr/WebXRManager.html

```diff
@@ -32,19 +32,25 @@ <h3>[property:Boolean isPresenting]</h3>
 
 		<h2>Methods</h2>
 
+		<h3>[method:ArrayCamera getCamera]()</h3>
+		<p>
+		Returns an instance of [page:ArrayCamera] which represents the XR camera of the active XR session.
+		For each view it holds a separate camera object in its [page:ArrayCamera.cameras cameras] property.
+		</p>
+
```

Comment:
This function [takes a camera object](https://github.com/mrdoob/three.js/blob/dev/src/renderers/webxr/WebXRManager.js#L348) which is used to position the XR camera and also [implicitly updates the matrix and matrixWorld properties of the passed camera](https://github.com/mrdoob/three.js/blob/cff4781347437fa62073aa85a732abbc41c833c0/src/renderers/webxr/WebXRManager.js#L380-L382), which is useful but a bit confusing. It also means that the xr camera is unnecessarily updated twice since this function is called during rendering which should be documented if it's going to be considered public, but perhaps that's not so bad.

Reply:
Ops, missed to document the parameter. Fixing...

Reply:
For now, I don't think it's necessary to provide the details for the parameter to the user. It's sufficient to document the signature and how the method should be used.